package.json

```diff
@@ -166,32 +199,26 @@
   ],
   "scripts": {
     "type-check": "tsc --noEmit --pretty",
-    "type-check:watch": "npm run type-check -- --watch",
+    "type-check:watch": "yarn run type-check -- --watch",
     "release": "standard-version -a",
-    "test:clean": "npx jest --clearCache",
+    "test:clean": "jest --clearCache",
     "test:acceptance": "codeceptjs run --steps",
-    "test:acceptance:server": "concurrently --kill-others \"npm run verdaccio:server\" \"npm run test:acceptance\"",
-    "test:e2e": "cross-env BABEL_ENV=test jest --config ./test/jest.config.e2e.js",
-    "test": "cross-env NODE_ENV=test BABEL_ENV=test TZ=UTC jest --config ./jest/jest.config.js --maxWorkers 2 --passWithNoTests",
-    "test:update-snapshot": "npm run test -- -u",
+    "test:acceptance:server": "concurrently --kill-others \"yarn run verdaccio:server\" \"yarn run test:acceptance\"",
+    "test:e2e": "cross-env jest --config ./test/jest.config.e2e.js",
+    "test": "cross-env NODE_ENV=test TZ=UTC jest --config ./jest/jest.config.js --maxWorkers 2 --passWithNoTests",
+    "test:update-snapshot": "yarn run test -- -u",
     "test:size": "bundlesize",
-    "lint": "npm run lint:js && npm run lint:css && npm run lint:lockfile",
-    "lint:js": "npm run type-check && eslint . --ext .js,.ts,.tsx",
-    "lint:html": "htmllinter src/template/index.html",
-    "lint:css": "stylelint \"src/**/styles.ts\"",
-    "lint:lockfile": "lockfile-lint --path yarn.lock --type yarn --validate-https --allowed-hosts verdaccio npm yarn",
+    "lint": "yarn run lint:js && yarn run lint:css",
+    "lint:js": "yarn run type-check && eslint . --ext .js,.ts,.tsx",
+    "lint:html": "yarn htmllinter src/template/index.html",
+    "lint:css": "yarn stylelint \"src/**/styles.ts\"",
     "coverage:publish": "codecov",
-    "pre:webpack": "rimraf static/*",
-    "dev:web": "cross-env BABEL_ENV=ui babel-node tools/dev.server.js",
+    "dev:web": "babel-node tools/dev.server.js",
```

Comment:
it looks much more clear :)

Reply:
The reason of those was because `@verdaccio/babel-preset`, so not need it anymore.